RestAPI Audit: Return code (404) of EFGS differs from documented return code (400) in document EU_Interop_Architectur

## Bug priority: 
Low

## Describe the bug

When in URL for audit RestAPI a wrong batchTag (e.g. 'wrongBatchTag') is used, the return code of EFGS (=404) differs from the return code (=400) documented in 20201408_EU_Interop_Architecture_v1.3_draft.pdf (page 45)

## Expected behaviour

Return code in document EU_Interop_Architecture and return code returned by EFGS should be the same.

## Steps to reproduce the issue

Use the following URL:
https://test-efgs-ws.tech.ec.europa.eu/diagnosiskeys/audit/download/2020-09-08/wrongBatchTag


## Possible Fix

Correction in documentation or in EFGS software.

## Additional context (Test details) ######################################################

```
auditDate = '2020-09-08', auditBatchTag = 'wrongBatchTag'

    @pytest.mark.parametrize('auditDate', [dateToday])
    @pytest.mark.parametrize('auditBatchTag', ['wrongBatchTag'])
    def test_invalid_batchTag(auditDate, auditBatchTag):
        headerToSet= {'Accept': 'application/json;version=1.0'}
        urlToSet= base_url + 'diagnosiskeys/audit/download/' + auditDate + '/' + auditBatchTag
    
        printReport.description("When for audit RestAPI a wrong batchTag is used, the EFGS should return an error code 400.")
        printReport.parameter('URL', urlToSet)
        printReport.headers(headerToSet)
    
        response = requests.get(url=urlToSet, headers=headerToSet, cert=cert)
        printReport.requestResponse_headerBody(response)
>       assert response.status_code == 400
E       assert 404 == 400
E        +  where 404 = <Response [404]>.status_code

tests\test_negTest_audit.py:28: AssertionError

```

--- a/docs/openapi.json
+++ b/docs/openapi.json
@@ -60,7 +60,7 @@
             }
           },
           "400": {
-            "description": "Invalid BatchTag used."
+            "description": "Invalid or missing request header."
           },
           "403": {
             "description": "Forbidden call in cause of missing or invalid client certificate."
@@ -256,7 +256,7 @@
             }
           },
           "400": {
-            "description": "Invalid BatchTag used."
+            "description": "Invalid or missing request header."
           },
           "403": {
             "description": "Forbidden call in cause of missing or invalid client certificate."

--- a/src/main/java/eu/interop/federationgateway/controller/AuditController.java
+++ b/src/main/java/eu/interop/federationgateway/controller/AuditController.java
@@ -96,7 +96,7 @@ public class AuditController {
     },
     responses = {
       @ApiResponse(responseCode = "200", description = "OK. Returns the audit information to the selected batch."),
-      @ApiResponse(responseCode = "400", description = "Invalid BatchTag used.", content = @Content),
+      @ApiResponse(responseCode = "400", description = "Invalid or missing request header.", content = @Content),
       @ApiResponse(responseCode = "403",
         description = "Forbidden call in cause of missing or invalid client certificate.", content = @Content),
       @ApiResponse(responseCode = "404", description = "BatchTag not found or no data exists.", content = @Content),

--- a/src/main/java/eu/interop/federationgateway/controller/DownloadController.java
+++ b/src/main/java/eu/interop/federationgateway/controller/DownloadController.java
@@ -114,7 +114,7 @@ public class DownloadController {
           mediaType = MediaType.APPLICATION_JSON_VALUE + "+v1.0",
           examples = @ExampleObject("diagnosisKeyBatch")
         )),
-      @ApiResponse(responseCode = "400", description = "Invalid BatchTag used.", content = @Content),
+      @ApiResponse(responseCode = "400", description = "Invalid or missing request header.", content = @Content),
       @ApiResponse(responseCode = "403",
         description = "Forbidden call in cause of missing or invalid client certificate.", content = @Content),
       @ApiResponse(responseCode = "404", description = "BatchTag not found or no data exists.", content = @Content),

